client/gulpfile.js

```diff
@@ -142,7 +142,8 @@ gulp.task('scss:compileCritical', () => {
       browsers: [
         'last 2 versions',
         'iOS 8'
-      ]
+      ],
+      grid: false
```

Comment:
I forgot to ask - do you have a screenshot of what this looks like in IE?

Reply:
![screen shot 2018-01-08 at 11 13 26](https://user-images.githubusercontent.com/6051896/34668386-096a7f7e-f465-11e7-9c80-3d2308adc780.png)